index.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./src/TerraPlugin');
```

Comment:
Did we need to compile the plugin?

Reply:
Nope, this is consumed by a node server, no compilation is needed.

Reply:
If you're curious, this plugin is consumed here: https://github.com/cerner/kaiju/pull/41. To try it out you will have to preview a workspace in the review app. You won't see much different from before, mostly this is breaking out things that were already defined into separate npm modules.